### DISCUSSION TITLE
CSS - Centering of thumbnails and scrollable container

### DISCUSSION THREAD

**Discussion by mrjonnyg** [fengyuanchen/viewerjs#494] [General]
Just a couple of smaller CSS issues which I realise are pretty n00b, but I'd be very grateful for some pointers.  I can't get the thumbnail gallery centered horizontally (the control form above centers nicely), it stays at about 1/3 from the left-hand side, no matter what I try.  I'd also like to have the container div provide a vertical overflow scrollbar, rather than it being on the body element.  Have tried various things to manipulate but with no joy.

![css](https://user-images.githubusercontent.com/86962991/126157797-de114dcc-9953-4bf1-a5c4-4a5b32aa5ad8.png)

Thanks in advance!

**Comment by fengyuanchen** [comment]
It is not an issue with the Cropper.js at all. 😂

Anyway, some related references for you:

- https://blog.hubspot.com/website/center-div-css
- https://www.w3schools.com/css/css_align.asp
- https://css-tricks.com/centering-css-complete-guide/
